package/abs/rancher-abs

```diff
@@ -187,15 +187,16 @@ detach_aliyun_disk() {
 init() {
     get_instance_id
 
-    log "\n[init] on ${TRANSCATION_SEQ}" /tmp/rancher_abs.log
+    rm -f /tmp/rancher_abs.log
+    log "\n[init] on ${TRANSCATION_SEQ}"
 
     local region_id=$(curl http://100.100.100.200/latest/meta-data/region-id 2>/dev/null)
```

Comment:
You have multiple places to retrieve region_id and zone_id. Put them into reuseable functions.